app/src/test/java/org/simple/clinic/facility/change/FacilityListItemBuilderTest.kt

```diff
@@ -77,4 +79,66 @@ class FacilityListItemBuilderTest {
             district = facilityWithNullStreet.district,
             state = facilityWithNullStreet.state)))
   }
+
+  @Test
+  fun `when user location is present then facilities nearby user should be correctly identified`() {
+    val userLocation = Coordinates(latitude = 51.919068, longitude = 17.647919)
```

Comment:
I sort of understand, but I don't agree completely. There can only be one definition of "nearby" and it'll always be dependent on distance. The formula used for computing the distance can differ. I'd only put the formula behind an abstraction. 

> This can then be mocked in tests so we don't have to worry about using real values in tests where they aren't required.

Agree with this

Reply:
Yeah, my intention was to avoid having to think about which `Coordinates` to provide in tests that don't require them.

Reply:
Sure, I will do this in my next PR

Reply:
Done here: https://github.com/simpledotorg/simple-android/commit/7cfe669a68b549860eb0df1c478839fa68390085